### PR TITLE
Add tasks route with sample tasks file

### PR DESCRIPTION
**Pull request by danielemden**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -73,7 +74,7 @@ func main() {
 		log.Printf("Could not connect to mongodb:", err)
 		return
 	}
-	Db := client.Database("remap")
+	Db := client.Database("remap_dev")
```
Comment (swillner) [inline main.go:77]: Make that as an additional command line paramete?

File: main.go [hunk 2]
```diff
@@ -164,6 +165,39 @@ func main() {
...
+
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
+		w.Write(body)
+
+	})
+
 	log.Println("Listening on", listenAddr, "...")
-	http.ListenAndServe(listenAddr, nil)
+	log.Fatal(http.ListenAndServe(listenAddr, nil))
```
Comment (swillner) [inline main.go:193]: Does it make sense to always log the result as fatal?

File: tasks.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+                    "minimumValue": 1, 
+                    "maximumValue": 10, 
+                    "step": 1}
+            },
+            {
+                "type": "completion"
+            }
+        ]
+    }
+]
```
Comment (swillner) [inline tasks.json:27]:
```suggestion
]

```

Comment (swillner) [reply tasks.json:27]: (add newline at end)

File: tasks.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
+// example task
```
Comment (swillner) [inline tasks.json:1]: Move to `example` directory? This file should be where the binary resides in the end, anyway.

File: main.go [hunk 5]
```diff
@@ -164,6 +165,39 @@ func main() {
...
+}
+
+func loadTasks() ([]byte, error) {
+	filename := "tasks.json"
+	body, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return nil, err
+	}
+	return body, nil
```
Comment (swillner) [inline main.go:202]:
```suggestion
	return ioutil.ReadFile(filename)
```

Comment (swillner) [reply main.go:202]: But what if the file is not found?

File: main.go [hunk 6]
```diff
@@ -164,6 +165,39 @@ func main() {
...
+			fmt.Fprintf(w, "Wrong method")
+			return
+		}
+
+		_, ok := auth(w, r, apiKey, authColl)
+		if !ok {
+			return
+		}
+
+		body, err := loadTasks()
```
Comment (swillner) [inline main.go:181]: Shouldn't those come from the database?

File: main.go [hunk 7]
```diff
@@ -164,6 +165,39 @@ func main() {
...
+		if err != nil {
+			w.WriteHeader(http.StatusNotFound)
```
Comment (swillner) [inline main.go:183]: Maybe return an empty list?

File: main.go (hunk 1 above)
Comment (danielemden) [reply main.go:77]: 👍

File: main.go (hunk 2 above)
Comment (danielemden) [reply main.go:193]: Ask the golang wiki: https://golang.org/doc/articles/wiki/

File: tasks.json (hunk 4 above)
Comment (danielemden) [reply tasks.json:1]: Good point

File: main.go (hunk 7 above)
Comment (danielemden) [reply main.go:183]: Like so: `w.Write([]byte("[]"))` ?

File: main.go [hunk 8]
```diff
@@ -164,6 +166,47 @@ func main() {
...
+			w.Write([]byte("[]"))
```
Comment (swillner) [inline main.go:185]:
```suggestion
			fmt.Fprint(w, "[]")
```

File: main.go (hunk 6 above)
Comment (swillner) [reply main.go:182]: I could make a PR on that after this one is merged

File: .github/workflows/go.yml [hunk 9]
```diff
@@ -33,5 +33,5 @@ jobs:
     - name: Build
       run: go build -v .
 
-    - name: Test
-      run: go test -v .
+    # - name: Test
```
Comment (swillner) [inline .github/workflows/go.yml:36]: You can undo these comments after #2

**Pull request merged**